### PR TITLE
removed use of "banned words" to comply with "Docs writing guidelines"

### PR DESCRIPTION
**Pull request by WakaJoekoe**
[The writing guidelines](https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#ban-these-8-words) specify a list of 8 banned words. I have edited some files so that they do not contain these banned words.

### CODE REVIEW

**Review by Calinou — CHANGES_REQUESTED** [review]
Looks good to me, but please remove the `class_packetpeer.rst` change as it will be lost when the class reference is updated.

Contributions to the class reference should be sent on the main Godot repository, not here. This is because the class reference in this repository is generated automatically based on the XML files in the main repository. See [Contribute to the class reference](https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html) in the documentation :slightly_smiling_face:

File: development/compiling/cross-compiling_for_ios_on_linux.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Cross-compiling for iOS on Linux
 .. highlight:: shell
 
 The procedure for this is somewhat complex and requires a lot of steps,
-but once you have the environment properly configured it will be easy to
+but once you have the environment properly configured, you can
```
Comment (skyace65) [inline development/compiling/cross-compiling_for_ios_on_linux.rst:9]:
```suggestion
but once you have the environment properly configured you can
```

File: tutorials/export/exporting_pcks.rst [hunk 2]
```diff
@@ -137,8 +137,8 @@ The PCK file contains a “mod_scene.tscn” test scene in its root.
 Summary
 -------
 
-This tutorial should illustrate how easy adding mods, patches or DLC to a game
-is. The most important thing is to identify how one plans to distribute future
+This tutorial should illustrate how to add mods, patches, or DLC to a game.
```
Comment (skyace65) [inline tutorials/export/exporting_pcks.rst:140]:
```suggestion
This tutorial explains how to add mods, patches, or DLC to a game.
```

File: tutorials/plugins/editor/making_main_screen_plugins.rst [hunk 3]
```diff
@@ -6,8 +6,8 @@ Making main screen plugins
 What this tutorial covers
 -------------------------
 
-As seen in the :ref:`doc_making_plugins` page, making a basic plugin that
-extends the editor is fairly easy. Main screen plugins allow you to create
+As seen in the :ref:`doc_making_plugins` page, you can make a basic plugin
+that extends the editor. Main screen plugins allow you to create
```
Comment (skyace65) [inline tutorials/plugins/editor/making_main_screen_plugins.rst:10]:
```suggestion
Main screen plugins allow you to create
```
I think for this the paragraph flows better if the sentence is removed entirely.

**Review by skyace65 — CHANGES_REQUESTED** [review]
Looks good so far. Rebasing should fix the build issue. Also once you're done these commits need to be squashed.

**Comment by mhilbrunner** [comment]
Salvaged to fix CI in https://github.com/godotengine/godot-docs/pull/6102, giving credit to @WakaJoekoe and applying the last suggestions above.

🎉 Thanks for contributing, congrats on your first contribution, and sorry the CI was a bit messed up :) It's fixed now!

**Pull request closed without merging**